src/proposals/20231011/SparkEthereum_20231011.t.sol

```diff
@@ -0,0 +1,194 @@
+            decimals:                 6,
+            ltv:                      0,
+            liquidationThreshold:     0,
+            liquidationBonus:         0,
+            liquidationProtocolFee:   0,
+            reserveFactor:            5_00,
+            usageAsCollateralEnabled: false,
+            borrowingEnabled:         true,
+            interestRateStrategy:     _findReserveConfigBySymbol(allConfigsAfter, 'USDT').interestRateStrategy,
+            stableBorrowRateEnabled:  false,
+            isPaused:                 false,
+            isActive:                 true,
+            isFrozen:                 false,
+            isSiloed:                 true,
+            isBorrowableInIsolation:  false,
+            isFlashloanable:          true,
+            supplyCap:                30_000_000,
+            borrowCap:                0,
+            debtCeiling:              0,
+            eModeCategory:            2
+        });
+        _validateReserveConfig(expectedUSDTConfig, allConfigsAfter);
+        _validateInterestRateStrategy(
+            _findReserveConfigBySymbol(allConfigsAfter, 'USDT').interestRateStrategy,
+            expectedUSDTConfig.interestRateStrategy,
+            InterestStrategyValues({
+                addressesProvider:             address(poolAddressesProvider),
+                optimalUsageRatio:             0.95e27,
+                optimalStableToTotalDebtRatio: 0,
+                baseStableBorrowRate:          SparkEthereum_20231011(payload).VARIABLE_RATE(),
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            SparkEthereum_20231011(payload).VARIABLE_RATE(),
+                variableRateSlope2:            0.2e27
+            })
+        );
+        _validateAssetSourceOnOracle(
+            poolAddressesProvider,
+            SparkEthereum_20231011(payload).USDT(),
+            SparkEthereum_20231011(payload).USDT_PRICE_FEED()
+        );
+
+        // USDC
+        // Performing a full market validation
+        // It should be the same as USDT market, with only difference in 'supplyCap' & 'priceFeed'
+        ReserveConfig memory expectedUSDCConfig = ReserveConfig({
+            symbol:                  'USDC',
+            underlying:               SparkEthereum_20231011(payload).USDC(),
+            aToken:                   address(0),  // Mock, as they don't get validated, because of the "dynamic" deployment on proposal execution
+            variableDebtToken:        address(0),  // Mock, as they don't get validated, because of the "dynamic" deployment on proposal execution
+            stableDebtToken:          address(0),  // Mock, as they don't get validated, because of the "dynamic" deployment on proposal execution
```

Comment:
These are not checked in the `_validateReserveConfig`, but they should be checked to be unchanged from the previous config in 3 separate asserts. The comment should actually be updated to say these are unchecked in the `_validateReserveConfig`.

Reply:
Done.